docs/csharp/whats-new/relationships-between-language-and-library.md

```diff
@@ -1,55 +1,21 @@
 ---
 title: The relationship between language features and library types | Microsoft Docs
 description: Language features often rely on library types for implementation. Understand that relationship.
-ms.date: 07/20/2017
+ms.date: 10/26/2023
 ---
 
 # Relationships between language features and library types
 
-The C# language definition requires a standard library to have certain
-types and certain accessible members on those types. The compiler generates
-code that uses these required types and members for many different language
-features. When necessary, there are NuGet packages that contain types
-needed for newer versions of the language when writing code for environments
-where those types or members have not been deployed yet.
+The C# language definition requires a standard library to have certain types and certain accessible members on those types. The compiler generates code that uses these required types and members for many different language features. For this reason, C# versions are supported only for the corresponding .NET version and newer. That ensures the correct runtime behavior and the availability of all required types and members.
```

Comment:
```suggestion
The C# language definition requires a standard library to have certain types and certain accessible members on those types. The compiler generates code that uses these required types and members for many different language features. For this reason, C# versions are supported only for the corresponding .NET version and newer. That ensures the correct run-time behavior and the availability of all required types and members.
```

Reply:
that's what I thought, but Acrolinx complains about it.